platform/ios/PolyPodApp/Features/FeatureStorage.swift

```diff
@@ -18,117 +44,101 @@ final class FeatureStorage: ObservableObject {
         }
         return URL(fileURLWithPath: "")
     }()
-    
-    lazy private var featureDirUrl: URL =
-        URL(string: featuresFileUrl.path) ?? URL(fileURLWithPath: "")
 
     init(dataProtection: DataProtection) {
         self.dataProtection = dataProtection
         setup()
     }
     
+    func featureForId(_ id: FeatureId) -> Feature? {
+        for category in categoriesListSubject.value {
+            for feature in category.features {
```

Comment:
Can't this be stored in a hash so that we don't have to run over all elements of the array?

Reply:
We could, but I would avoid having two `sources` which need to be kept in sync. I would employ a hash for the means of performance, but we are more than ok with that here.